src/cockpit/389-console/src/lib/security/certificateManagement.jsx

```diff
@@ -192,44 +280,45 @@ export class CertificateManagement extends React.Component {
         });
     }
 
-    addCert () {
-        if (this.state.certName == "") {
-            this.props.addNotification(
-                "warning",
-                `Missing certificate nickname`
-            );
-            return;
-        } else if (this.state.certFile == "") {
-            this.props.addNotification(
-                "warning",
-                `Missing certificate file name`
-            );
-            return;
-        }
+    showExportModal (nickname) {
+        this.setState({
+            showExportModal: true,
+            exportNickname: nickname,
+            exportDERFormat: false,
+            exportFileName: "",
+        });
+    }
+
+    closeExportModal () {
+        this.setState({
+            showExportModal: false,
+            exportNickname: "",
+        });
+    }
 
+    exportCert () {
         this.setState({
             modalSpinning: true,
-            loading: true,
         });
-        const cmd = [
+        let cmd = [
             "dsconf", "-j", "ldapi://%2fvar%2frun%2fslapd-" + this.props.serverId + ".socket",
-            "security", "certificate", "add", "--name=" + this.state.certName, "--file=" + this.state.certFile
+            "security", "export-cert", this.state.exportNickname, "--output-file=" + this.state.exportFileName
```

Comment:
If I specify `exportFileName` as a "test_cert" - it will name the file exactly like that. Hence, making it undiscoverable by the UI later.

Should we additionally add `.crt` extension there if the user hasn't specified it?

Reply:
Fixed